aiowebostv/models.py

```diff
@@ -0,0 +1,53 @@
+"""LG webOS TV models."""
+
+from dataclasses import MISSING, dataclass, field, fields
+from typing import Any
+
+WebOsTvStateValue = dict[str, Any] | str | bool | int | list[dict[str, Any]] | None
+
+
+@dataclass
+class WebOsTvInfo:
+    """Represent LG webOS TV info."""
+
+    hello: dict[str, Any] = field(default_factory=dict[str, Any])
+    system: dict[str, Any] = field(default_factory=dict[str, Any])
+    software: dict[str, Any] = field(default_factory=dict[str, Any])
+
+    def clear(self) -> None:
+        """Reset all fields to their default values."""
+        for f in fields(self):
+            if f.default_factory is not MISSING:
+                setattr(self, f.name, f.default_factory())
+
+
+@dataclass
+class WebOsTvState:
+    """Represent the state of a LG webOS TV."""
+
+    power_state: dict[str, Any] = field(default_factory=dict[str, Any])
```

Comment:
```suggestion
    power_state: dict[str, Any] = field(default_factory=dict)
```

I think the typing redundant here